ci/kokoro/macos/build-quickstart-bazel.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+if [[ $# -ne 1 ]]; then
+  echo "Usage: $(basename "$0") <project-root>"
+  exit 1
+fi
+
+readonly PROJECT_ROOT="$1"
+
+source "${PROJECT_ROOT}/ci/colors.sh"
+source "${PROJECT_ROOT}/ci/etc/integration-tests-config.sh"
+source "${PROJECT_ROOT}/ci/etc/quickstart-config.sh"
+
+echo
+echo "================================================================"
+log_yellow "update or install Bazel."
+
+# macOS does not have sha256sum by default, but `shasum -a 256` does the same
+# thing:
+function sha256sum() { shasum -a 256 "$@"; } && export -f sha256sum
```

Comment:
Do we need the `&&` here w/ the export after? Or can the export just be on the following line. Maybe I'm asking, can the declaration of the function fail such that `&&` is needed?

Reply:
Removed.